docs/usage/cfcli/application_delete.md

```diff
@@ -2,7 +2,7 @@
 
 To get a list of existing apps deployed in the curent space run `cf apps`.
 
-``` bash
+``` language-html
```

Comment:
I assume there's a good reason for this but this block isn't html so I'm curious why the change?